code/modules/client/preference/character.dm

```diff
@@ -441,19 +451,21 @@
 	organ_data = params2list(query.item[42])
 	rlimb_data = params2list(query.item[43])
 	nanotrasen_relation = query.item[44]
-	speciesprefs = text2num(query.item[45])
+	physique = query.item[45]
+	height = query.item[46]
+	speciesprefs = text2num(query.item[47])
 
 	//socks
-	socks = query.item[46]
-	body_accessory = query.item[47]
-	loadout_gear = query.item[48]
-	autohiss_mode = text2num(query.item[49])
+	socks = query.item[48]
+	body_accessory = query.item[49]
+	loadout_gear = query.item[50]
+	autohiss_mode = text2num(query.item[51])
 	// Index [50] is the slot
-	h_grad_style = query.item[51]
-	h_grad_offset_x = query.item[52] // parsed down below
-	h_grad_colour = query.item[53]
-	h_grad_alpha = query.item[54]
-	var/custom_emotes_tmp = query.item[55]
+	h_grad_style = query.item[52]
```

Comment:
Offset this from 53 and update the above comment

Reply:
should be fixed, will resolve once i get the all clear